notebooks/hw-02-pure-pursuit-control.md

```diff
@@ -32,45 +32,47 @@ Starting udem-fall19_novnc_1        ... done
 Recreating udem-fall19_lanefollow_1 ... done
 ```
 
-Note that thesimulator is now running in its own container. 
+Note that the simulator now runs in its own container. 
 
 You can open the notebook just like before by copying the url that looks like:
 
-    http://127.0.0.1:8888/?token=<aaaaannnnn123123123123....>
+    http://127.0.0.1:8888/?token={SOME_LONG_TOKEN}
 
 into your browser. 
 
-You no longer need to build the `custom_ws` only the `catkin_ws`:
+You no longer need to build the `custom_ws`, but rather only the `catkin_ws`:
 
     $ catkin build --workspace catkin_ws
     $ source catkin_ws/devel/setup.bash
     
-If this is your first time it will take ~5 mins or so but since we mount things the build artifacts are preserved.
-You also need to manually start the "car interface" which runs the inverse kinematics.
+If this is the initial build, it will take around five minutes; Due to the `docker` mounts, the build artifacts will persist, so subsequent builds will be much faster.
+
+You also need to manually start the `car_interface`, which runs the inverse kinematics.
+
 You can do this with:
 
     $ ./launch_car_interface.sh
 
-Now you can run any demo in the duckietown codebase. Let's test out one that you have experience with already:
+Now you can run any demo in the Duckietown codebase. Let's test out one that you have experience with already:
 
     $ roslaunch duckietown_demos lane_following.launch
 
-which is the launch file that is run when you executed `dts duckiebot demo --package_name duckietown_demos --demo_name lane_following` in the last hardware exercise (now you can see what the `package_name` and `demo_name` flags do...).
+which is the launch file that was run when you previously executed `dts duckiebot demo --package_name duckietown_demos --demo_name lane_following` in the last hardware exercise (Here's a good way to see what the `package_name` and `demo_name` flags were doing with the old workflow).
```

Comment:
this is still what those flags are doing. That's what happens when you run dts duckiebot demo.